.github/actions/build-single-project/action.yml

```diff
@@ -33,7 +33,9 @@ runs:
       run: echo "yes" | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --install "cmake;3.22.1"
     - name: "Install NDK"
       shell: bash
-      run: echo "yes" | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --install "ndk;23.1.7779620"
+      run: |
+        NDK_VERSION=$(grep "ndkVersion" settings.gradle | awk -F "=" '{gsub(/"| /, ""); print $2}')
```

Comment:
Can you fail hard if it fails to find a reasonable value?

Reply:
Won't it fail when it tries to run `sdkmanager` with the bad value? I added `set -x` for logging as well.